src/Module.ts

```diff
@@ -267,7 +263,7 @@ export default class Module {
 			nodeConstructors,
 			propertyReadSideEffects:
 				!this.graph.treeshake || this.graph.treeshakingOptions.propertyReadSideEffects,
-			reassignmentTracker: new EntityPathTracker(),
+			reassignmentTracker,
```

Comment:
We already read things off graph here, so I would suggest just setting this as `reassignmentTracker: graph.reassignmentTracker` instead of a new argument, unless we switch all to being provided as arguments.

Reply:
Done